netty/src/test/java/io/grpc/netty/NettyClientTransportTest.java

```diff
@@ -284,8 +284,8 @@ public void maxHeaderListSizeShouldBeEnforcedOnServer()  throws Exception {
           + " size limit!");
     } catch (Exception e) {
       Throwable rootCause = getRootCause(e);
-      assertTrue(rootCause.getMessage(),
-          rootCause.getMessage().contains("Header size exceeded max allowed size (1)"));
+      assertTrue(rootCause instanceof StatusException);
```

Comment:
@nmittler very easy. One line in NettyClientHandler and one line in NettyServerHandler. It's just that exceeding the limit would kill the whole connection. That will be fixed in Netty 4.1.7 though netty/netty#5919.